jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -120,9 +120,21 @@ def _on_viewer_added(self, msg):
         self._create_viewer_callbacks(self.app.get_viewer_by_id(msg.viewer_id))
 
     def _on_global_display_unit_changed(self, msg):
-        # eventually should observe change in flux OR angle
-        if msg.axis == "flux":
-            self.image_unit = u.Unit(msg.unit)
+
+        # only concerned with handling unit changes for mouseover in cubeviz,
+        # until implemented in other configs
+        if self.config == 'cubeviz':
```

Comment:
I think this can be removed entirely for now and leave any other special-handling up to @gibsongreen's effort to enable in the other configs.  _Hopefully_ we can write the logic in a way that we don't need per-config handling 🤞

Reply:
This is one of the spots where I added the older logic into #3139, I can test this PR with and without to see the implications, I think the only config that could need specific handling (not necessarily here) is Imviz: https://github.com/spacetelescope/jdaviz/blob/e29478e83b9fc5cd4568f0b5c290f67e80e7e7df/jdaviz/configs/imviz/plugins/coords_info/coords_info.py#L124

Reply:
it will for sure need to be changed for imviz, i can't just intercept a sb changed message for imviz. i can remove it for now though i think?